bumble/device.py

```diff
@@ -1181,12 +1191,19 @@ def find_connection_by_bd_addr(
         return None
 
     def create_l2cap_connector(self, connection, psm):
+        warnings.warn('Please use l2cap_channel_manager.connect', DeprecationWarning)
         return lambda: self.l2cap_channel_manager.connect(connection, psm)
 
     def create_l2cap_registrar(self, psm):
+        warnings.warn(
```

Comment:
I think we can just get rid of this, it is only used in one place internally, and I doubt there are many external callers.